zipkin-lens/package.json

```diff
@@ -68,8 +68,8 @@
     "@material-ui/core": "^4.1.1",
     "@material-ui/pickers": "^3.1.1",
     "@material-ui/styles": "^4.1.1",
-    "babel-polyfill": "^6.26.0",
```

Comment:
`@babel/polyfill` is deprecated.
Please see the top of this page (https://babeljs.io/docs/en/babel-polyfill)